### ISSUE TITLE
Mismatch between conservation status display on website and in downloaded csv file

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/Pantheon#282]
The species Teuchophorus nigricosta is shown on the website as having no conservation status, as as being associated with "Arthropoda, Fagales, Pinus, Plantae":

![image](https://user-images.githubusercontent.com/16039111/197421183-0508b6c5-3d83-49a7-a78a-3562f319ab53.png)

However, in the csv download of the Pantheon species index, there is a conservation status of "LC;NS" and the association is given as "Arthropoda, Fagales".

I think the website has the correct status (i.e. no status), but the website seems to be incorrect for the association (there i no link to Pinus or Plantae as far as I'm aware). And more generally of course it is worrying that there are mismatches at all.

**Comment by kitenetter** [comment]
Another example: the moth Ennomos fuscantaria has status "NT;Section 41 Priority Species - research only" on the website, but "None;Section 41 Priority Species - research only" in the csv. Is the csv download outputting a different version of the database than is being used to populate the website?

**Comment by kitenetter** [comment]
There are lots more examples, so it does appear that the csv download of the entire species dictionary is outputting an old version. The download is the one available from the species index page:
https://pantheon.brc.ac.uk/species-index

**Comment by kitenetter** [comment]
The mismatch between the website and the csv download has been resolved since this issue was raised.

The species index can be downloaded in three places:

1. [Species Index download](https://pantheon.brc.ac.uk/content/species-index-download) page
2. [Species index](https://pantheon.brc.ac.uk/species-index) page: download link at top of the page
3. [Species index](https://pantheon.brc.ac.uk/species-index) page: download link at bottom of the page

The first and third of those links is outputting a correct file; the second link was outputting an incorrect file (e.g. there were old synonyms in it instead of current recommended names), so I have edited the link that used to be there, and instead pointed it back to the separate download page (item 1 in the above list).

Tagging @johnvanbreda for info in case any other action needs to be taken in relation to the link that I removed.

Closing.
